options/locale/locale_en-US.ini

```diff
@@ -1007,6 +1008,8 @@ delete_preexisting_success = Deleted unadopted files in %s
 blame_prior = View blame prior to this change
 author_search_tooltip = Shows a maximum of 30 users
 
+tree_path_not_found = Path %[1]s doesn't exist in branch %[2]s
```

Comment:
Error message doesn't work when visiting an unknown tree path for a commit or tag instead of a branch.